test/boostTest.cpp

```diff
@@ -83,6 +83,17 @@ int main( int argc, char* argv[] )
 {
 	//Initialize options
 	dev::test::Options const& opt = dev::test::Options::get(argc, argv);
+
+	char* path = (argv[0]);
+	argc = 3;
```

Comment:
We should not just ignore the options that are passed here. Also I think it is really bad style to modify `argv`.